modules/tinymce/src/core/main/ts/api/OptionTypes.ts

```diff
@@ -109,6 +109,7 @@ interface BaseEditorOptions {
   forced_root_block?: string;
   forced_root_block_attrs?: Record<string, string>;
   formats?: Formats;
+  format_wrap_noneditable_selectors?: string[];
```

Comment:
Since this is largely aimed at noneditable, might I suggest: `noneditable_format_wrap_selector`.

Also this option type is inconsistent and having an array doesn't make sense so this should just be a single string.

- [ ] Change option to be a single selector for consistency

Reply:
Also as this has a default, we should declare this as non-optional in the `EditorOptions` type. For example, `format_wrap_noneditable_selectors: string`

Reply:
Another name that might work? `format_noneditable_selector` I know that drops the wrap, but it's not very clear what that applies to just from the name so I think we'd be better clarifying that via docs if needed instead 🤷

Reply:
I think I like `format_noneditable_selector` the best so am happy to go with that.

Reply:
The option name and return type has been updated